src/main/java/com/box/sdk/BoxJSONObject.java

```diff
@@ -192,7 +197,7 @@ void update(JsonObject jsonObject) {
 
             this.parseJSONMember(member);
         }
-
+        this.jsonObject = jsonObject;
```

Comment:
Any benefit to moving this up to the first line of the method?  I'm thinking that we'd gain the ability to save the jsonObject even if parseJSONMember throws an exception.

Reply:
Whenever update is called manually, the object will be saved if we put it update. If we move it up, it will only be called when instantiated.

Reply:
No, I'm not suggesting to move it out of the method.  See my code suggestion...